portality/formcontext/forms.py

```diff
@@ -249,6 +249,18 @@ class JournalInformation(Form):
         [OptionalIf('publishing_rights', optvals=Choices.publishing_rights_url_optional()), URLOptionalScheme()]
     )
 
+class JournalInfoOptionalPaymentURLs(JournalInformation):
```

Comment:
The name really drives home the point of the complexity :). Good thing we just did a major refactor..